tests/phpunit/tests/media.php

```diff
@@ -3798,6 +3798,83 @@ function() use ( &$result ) {
 
 		$this->assertFalse( $result );
 	}
+
+	/**
+	 * @ticket 55443
```

Comment:
```suggestion
	 * @ticket 55443
	 *
	 * @covers ::wp_print_image_mime_fallback_script
```
Same for all new test methods in the PR.